src/System.Net.HttpListener/tests/HttpRequestStreamTests.cs

```diff
@@ -313,7 +313,6 @@ public void Dispose()
         }
 
         [Fact]
-        [ActiveIssue(18128, platforms: TestPlatforms.AnyUnix)] // NotSupportedException thrown instead of InvalidOperationException
```

Comment:
It fixes another assertion failure in the test method, There were multiple issues here: https://github.com/hughbe/corefx/blob/b3b47377337e1cb5532443fc63326d82b91a518e/src/System.Net.HttpListener/tests/HttpRequestStreamTests.cs#L329